openedx/features/edly/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+from django.db import models
+from model_utils.models import TimeStampedModel
+from organizations.models import Organization
+
+ALPHANUMERIC_VALIDATOR = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+
+class EdlyOrganization(TimeStampedModel):
+    """
+    EdlyOrganization model for Edly.
+    """
+    name = models.CharField(max_length=255)
+    slug = models.CharField(max_length=100, unique=True, validators=[ALPHANUMERIC_VALIDATOR])
+
+    def __str__(self):
+        return '{name}: ({slug})'.format(name=self.name, slug=self.slug)
+
+
+class EdlySubOrganization(TimeStampedModel):
```

Comment:
@asadmanzoor93 this model should have a slug field too.

Reply:
As per the previous review suggestion, we removed initially added `slug` and `name` fields for `EdlySubOrganization`. Kindly suggest should we have both fields or only the `slug` field.

Reply:
This feedback is now resolved by adding `slug` and `name` fields for `EdlySubOrganization`.